### DISCUSSION TITLE
Local subtitle translation with LibreTranslate

### DISCUSSION THREAD

**Discussion by 0xT3chN0** [McCloudS/subgen#108] [Ideas]
Hi, 

Would it be possible, to add LibreTranslate for automated translation and to generate multiple versions of subtitles?
Whisper is currently only capable of translating Audio to ENG Sub. 
LibreTranslate is quite powerful and is also self hostable.

I found a project that already has this feature built-in, also with a Web-UI, but it can't receive tasks from Plex or Bazarr, like SubGen can: https://github.com/pluja/whishper

Have a nice day.

**Comment by McCloudS** [comment]
While certainly possible, it isn't a use-case for me, so it's not planned.  There are ways to 'trick' the model to translate for you if you look at https://github.com/McCloudS/subgen/issues/104

You can also use the translate option in Bazarr.
